Modules/TestQuestionPool/classes/class.assFileUpload.php

```diff
@@ -1177,18 +1177,14 @@ protected function isFileReuseHandlingRequired(): bool
      */
     protected function isFileUploadAvailable(): bool
     {
-        if (!isset($_FILES['upload'])) {
-            return false;
-        }
-
-        if (!isset($_FILES['upload']['tmp_name'])) {
+        if(!$this->file_upload->hasUploads()) {
```

Comment:
```suggestion
        if (!$this->file_upload->hasUploads()) {
```

Reply:
Thanks for pointing that out, @mjansenDatabay. I've made the necessary adjustments to the code.